### ISSUE TITLE
Improved example documentation

### ISSUE THREAD

**Issue by TedGoas** [pshihn/rough-notation#35]
Hey thanks so much for making this, looks fantastic. I'm having a little trouble getting started because I'm a JS novice. I'd really appreciate anything that makes this documentation more approachable for folks like me.

For example, after getting the library into my site, I'm not sure where I should be including JavaScript for each instance. Or the best way to include multiple instances on a page. The documentation isn't specific and the examples are hard to follow by viewing source. Any advice you can offer would be great.

<img width="931" alt="Screenshot 2020-06-08 20 46 07" src="https://user-images.githubusercontent.com/1172461/84093832-5488e600-a9c9-11ea-83e6-8d40b659b9b6.png">
(I'm not sure where this goes)

**Comment by pshihn** [comment]
I've created basic examples to play with here: https://github.com/pshihn/rough-notation/blob/master/README.md#live-examples

**Comment by TedGoas** [comment]
Fantastic, many thanks!
